src/openrct2/peep/Guest.cpp

```diff
@@ -4987,8 +4990,9 @@ void Guest::UpdateRideLeaveExit()
 
     if (ride != nullptr && (PeepFlags & PEEP_FLAGS_TRACKING))
     {
-        auto nameArgLen = FormatNameTo(gCommonFormatArgs);
-        ride->FormatNameTo(gCommonFormatArgs + nameArgLen);
+        auto ft = Formatter::Common();
+        FormatNameTo(ft);
+        ride->FormatNameTo(gCommonFormatArgs + ft.NumBytes());
```

Comment:
As above

`ride->FormatNameTo(ft);`

Reply:
I didn't want to touch to the ride class yet in this PR.